File: LB/PROgram0043.c
/*

Q43) Write a PROgram which Accept number from user and return count of digits if they are between 3 to 7

    input :- 2395
    output :- 2

*/

#include<stdio.h>

int Check3to7(int iNo1)
{
    int iCnt = 0;
    int iDigit = 0;

    for(; iNo1 != 0;)
    {
        iDigit = iNo1 % 10;

        if((iDigit >= 3) && (iDigit <= 7))
        {
            iCnt++;
        }

        iNo1 = iNo1 / 10;
    }

    return iCnt;
}

int main()
{
    int iValue1 = 0;
    int iRet = 0;

    printf("Enter the Number : ");
    scanf("%d",&iValue1);

    iRet = Check3to7(iValue1);

    printf("Count of Digits which are between 3 to 7 is : %d",iRet);

    return 0;
}

File: PM/PROgram0043.py
'''

Q3) Write a PROgram that Print Square of number whch is Accepted from user

'''

def ChkSquare(iNo1):
    iAns = 0
    iAns = iNo1 * iNo1
    return iAns

def main():
    print("Enter the Number : ")
    iValue1 = int(input())

    iRet = ChkSquare(iValue1)

    print("Square of Given Number is : ",iRet)    

if __name__ == "__main__":
    main()        